configure.ac

```diff
@@ -282,7 +282,7 @@ AC_CACHE_CHECK([where ${PROTOC} sources its headers from],
                        nl_maybe_protoc_includes="${PROTOC_INCLUDES}"
 
                    else
-                       protoc_dirname=`dirname $(which ${PROTOC})`
+                       protoc_dirname=`dirname $(readlink -f ${PROTOC})`
```

Comment:
This works because `PROTOC` is expected to contain the absolute path as it is set by default via AC_PATH_PROG.  I think this argues that it would be an error to pass in a relative path `PROTOC` variable on the commandline to configure.  If we want to permit the relative paths for `PROTOC`, the above could be modified to read `$(readlink -f $(which ${PROTOC}))`

Reply:
Fixed in the most recent revision with a Python-equivalent of readlink -f.